packages/frameworks/vue/src/presence/presence.props.ts

```diff
@@ -1,21 +1,8 @@
-import type { Context } from '@zag-js/presence'
-import type { PropType } from 'vue'
-import { declareEmits } from '../utils'
-import type { UsePresenceProps } from './use-presence'
-
 export const props = {
   present: {
-    type: Boolean as PropType<Context['present']>,
+    type: Boolean,
     default: undefined,
   },
-  lazyMount: {
-    type: Boolean as PropType<UsePresenceProps['lazyMount']>,
-    default: false,
-  },
-  unmountOnExit: {
-    type: Boolean as PropType<UsePresenceProps['unmountOnExit']>,
-    default: false,
-  },
+  lazyMount: Boolean,
+  unmountOnExit: Boolean,
 }
-
-export const emits = declareEmits(['exit-complete'])
```

Comment:
As we use Function Signature to define component, I don't think we need to declare emit to get the type hint of event, because they already exist in the definition of zag